src/lambda_extract_metrics.py

```diff
@@ -8,7 +8,7 @@
 from lib.settings import Settings
 from lib.core.constants import SettingConfigs
 
-from lib.metrics_extractor import MetricsExtractorProvider
+from lib.metrics_extractor import MetricsExtractorProvider, get_last_update_time
```

Comment:
I would not import the functions like classes, because it is harder to find where this function comes from (for classes it is obvious since file name is the same as class name). 

```
from lib.metrics_extractor import MetricsExtractorProvider
from lib.metrics_extractor.metrics_extractor_utils import get_last_update_time
```

Reply:
fixed, thanks